recipe/bld.bat

```diff
@@ -3,9 +3,11 @@ mkdir build && cd build
 
 set CMAKE_CONFIG="Release"
 
+REM nlopt is used by r-nloptr so a Python interface isn't needed on win64
 cmake -LAH -G"NMake Makefiles"                     ^
   -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%"           ^
   -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"        ^
+  -DNLOPT_PYTHON=OFF ^
```

Comment:
@skupr-anaconda can you explain why we don't want the Python interfaces for `win-64`?

Reply:
Every supported programming language can have its interface of `nlopt` but creating the **Python** interface will require `python` in `host` and at `runtime`. That means then we need to build `nlopt` for all `python` versions (**four** at this moment). Hence, `r-nloptr` will be built for all 4 Python versions and the same applies to all 350 downstream R packages. It's a lot of redundancy. And R packages seem not to require the Python interface.